src/app/payment/paymentMethod/StripeV3CustomCardForm.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import { mount } from 'enzyme';
+import React, { FunctionComponent } from 'react';
+
+import StripeV3CustomCardForm, { StripeV3CustomCardFormProps } from './StripeV3CustomCardForm';
+
+describe('StripeV3CustomCardForm', () => {
+    let defaultProps: StripeV3CustomCardFormProps;
+    let StripeV3CustomCardFormTest: FunctionComponent<StripeV3CustomCardFormProps>;
+
+    beforeEach(() => {
+        defaultProps = {
+            options: {
+                cardNumberElementOptions: {
+                    containerId: 'stripe-number-component-field',
```

Comment:
💅🏻  Could you please rename this container to `stripe-card-number-component-field`?